src/makeRobot.js

```diff
@@ -38,7 +38,54 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    x: 0,
```

Comment:
[Task] The coordinates of the robot must be stored in the object coords, the keys x and y inside the robot.
```suggestion
    coords: {
      x: 0,
      y: 0,
    },
```